Gems/AtomLyIntegration/EMotionFXAtom/Code/Tools/EMStudio/AnimViewportRenderer.h

```diff
@@ -91,6 +94,8 @@ namespace EMStudio
         const RenderOptions* m_renderOptions;
 
         const float DefaultFrustumDimension = 128.0f;
+        const float BoundMaxDistance = 150.0f;
```

Comment:
That's 150 from 500 tiles, don't we see the ground plane popping in the horizon with such a high number? What about doing something like 10 or so? Personally, I'd remove the condition entirely and just re-position it every frame.